src/components/AboutSettingsList/AboutSettingsList.tsx

```diff
@@ -0,0 +1,91 @@
+import { BlueBase, BlueBaseContext, IntlConsumer, IntlContextData, Theme } from '@bluebase/core';
+import { Divider, Icon, List, View } from '@bluebase/components';
+import { I18nManager, TextStyle } from 'react-native';
+
+import { ExternalLink } from '../ExternalLink';
+import React from 'react';
+
+export interface AboutSettingsListStyles {
+	iconRight: TextStyle;
+}
+
+export interface AboutSettingsListProps {
+	styles?: Partial<AboutSettingsListStyles>;
+}
+
+export class AboutSettingsList extends React.PureComponent<AboutSettingsListProps> {
+
+			static contextType = BlueBaseContext;
+
+			static defaultStyles = (theme: Theme) => ({
+			iconRight: {
+			color: theme.palette.text.disabled,
+			transform: [{ scaleX: I18nManager.isRTL ? -1 : 1 }],
+		},
+		})
+
+			renderVersion = ({ __ }: IntlContextData) => {
+			const BB: BlueBase = this.context;
+			const version = BB.Configs.getValue('version');
+
+			return version && <List.Item title={__('Version')} description={__(version)} />;
+		}
+
+			renderAuthor = ({ __ }: IntlContextData) => {
+			const BB: BlueBase = this.context;
+			const { styles = {} } = this.props;
+
+			const author = BB.Configs.getValue('author');
```

Comment:
can we add a default author in configs?

Reply:
yes we can  but what who would be the default author and why we want to do this